### PR TITLE
Added DOI to readme.

### PR DESCRIPTION
**Pull request by rrsettgast**
Generated a pre-release to get a DOI. This adds it to our readme.

### CODE REVIEW

**Review by klevzoff — APPROVED** [review]

**Comment by klevzoff** [comment]
We might also want to add a [citation file](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-citation-files)

**Review by castelletto1 — APPROVED** [review]

**Review by herve-gross — APPROVED** [review]

**Comment by rrsettgast** [comment]
@joshua-white @castelletto1 @francoishamon @klevzoff  @CusiniM 

I added a draft of a `CITATION.cff` file. I just grabbed the contributor list. The ordered the top...10 or so. The rest are just placed. Let me know if I have left anyone out, or if you feel I need to change order.

**Review by jhuang2601 — APPROVED** [review]

**Review by CusiniM — APPROVED** [review]

**Comment by joshua-white** [comment]
Should we make an effort to actually get everyone's ORCID?

**Comment by rrsettgast** [comment]
> Should we make an effort to actually get everyone's ORCID?

I think we should have folks update their own ORCID

File: CITATION.cff [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+authors:
+- family-names: "Settgast"
+  given-names: "Randolph Richard"
+  orcid: "https://orcid.org/0000-0002-2536-7867"
+- family-names: "Corbett"
+  given-names: "Benjamin Curtice"
+  orcid: "https://orcid.org/0000-0000-0000-0000"
+- family-names: "Klevtsov"
+  given-names: "Sergey"
+  orcid: "https://orcid.org/0000-0000-0000-0000"
```
Comment (klevzoff) [inline CITATION.cff:12]:
```suggestion
  orcid: "https://orcid.org/0000-0001-9044-1827"
```

File: CITATION.cff [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+- family-names: "Borio"
+  given-names: "Andrea"
+  orcid: "https://orcid.org/0000-0000-0000-0000"
+- family-names: "Franceschini"
+  given-names: "Andrea"
+  orcid: "https://orcid.org/0000-0000-0000-0000"
+- family-names: "Huang"
+  given-names: "Jian"
+  orcid: "https://orcid.org/0000-0000-0000-0000"
```
Comment (jhuang2601) [inline CITATION.cff:42]:
```suggestion
  orcid: "https://orcid.org/0000-0002-5380-2563"
```

File: CITATION.cff [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+- family-names: "Han"
+  given-names: "Brian"
+  orcid: "https://orcid.org/0000-0000-0000-0000"
+- family-names: "Gross"
+  given-names: "Herve"
+  orcid: "https://orcid.org/0000-0000-0000-0000"
```
Comment (herve-gross) [inline CITATION.cff:48]: orchid: "https://orcid.org/0000-0002-1747-2018"

Comment (herve-gross) [inline CITATION.cff:48]:
```suggestion
  orcid: "https://orcid.org/0000-0002-1747-2018"
```

File: CITATION.cff [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+- family-names: "Gazzola"
+  given-names: "Thomas"
+  orcid: "https://orcid.org/0000-0000-0000-0000"
+- family-names: "Tobin"
+  given-names: "William"
+  orcid: "https://orcid.org/0000-0000-0000-0000"
+- family-names: "White"
+  given-names: "Joshua"
+  orcid: "https://orcid.org/0000-0000-0000-0000"
```
Comment (joshua-white) [inline CITATION.cff:30]:
```suggestion
  orcid: "https://orcid.org/0000-0003-3491-142X"
```

File: CITATION.cff [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+  orcid: "https://orcid.org/0000-0002-2536-7867"
+- family-names: "Corbett"
+  given-names: "Benjamin Curtice"
+  orcid: "https://orcid.org/0000-0000-0000-0000"
+- family-names: "Klevtsov"
+  given-names: "Sergey"
+  orcid: "https://orcid.org/0000-0000-0000-0000"
+- family-names: "Hamon"
+  given-names: "Francois"
+  orcid: "https://orcid.org/0000-0000-0000-0000"
```
Comment (francoishamon) [inline CITATION.cff:15]:
```suggestion
  orcid: "https://orcid.org/0000-0001-8229-963X"
```

File: CITATION.cff [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+- family-names: "Tobin"
+  given-names: "William"
+  orcid: "https://orcid.org/0000-0000-0000-0000"
+- family-names: "White"
+  given-names: "Joshua"
+  orcid: "https://orcid.org/0000-0003-3491-142X"
+- family-names: "Castelletto"
+  given-names: "Nicola"
+  orcid: "https://orcid.org/0000-0000-0000-0000"
```
Comment (castelletto1) [inline CITATION.cff:33]:
```suggestion
  orcid: "https://orcid.org/0000-0001-6816-6769"
```

File: CITATION.cff [hunk 7]
```diff
@@ -0,0 +1,92 @@
...
+- family-names: "Borio"
+  given-names: "Andrea"
+  orcid: "https://orcid.org/0000-0000-0000-0000"
```
Comment (castelletto1) [inline CITATION.cff:36]:
```suggestion
  orcid: "https://orcid.org/0000-0003-2016-5403"
```

File: CITATION.cff [hunk 8]
```diff
@@ -0,0 +1,92 @@
...
+- family-names: "Franceschini"
+  given-names: "Andrea"
+  orcid: "https://orcid.org/0000-0000-0000-0000"
```
Comment (castelletto1) [inline CITATION.cff:39]:
```suggestion
  orcid: "https://orcid.org/0000-0003-4395-5125"
```

File: CITATION.cff [hunk 9]
```diff
@@ -0,0 +1,92 @@
...
+- family-names: "Citrain"
+  given-names: "Aurelien"
+  orcid: "https://orcid.org/0000-0000-0000-0000"
+- family-names: "Vargas"
+  given-names: "Arturo"
+  orcid: "https://orcid.org/0000-0000-0000-0000"
+- family-names: "Cremon"
+  given-names: "Matthias"
+  orcid: "https://orcid.org/0000-0000-0000-0000"
```
Comment (castelletto1) [inline CITATION.cff:63]:
```suggestion
  orcid: "https://orcid.org/0000-0001-7458-6401"
```

File: CITATION.cff [hunk 10]
```diff
@@ -0,0 +1,92 @@
...
+  orcid: "https://orcid.org/0000-0001-8229-963X"
+- family-names: "Sherman"
+  given-names: "Christopher"
+  orcid: "https://orcid.org/0000-0003-3550-0657"
+- family-names: "Cusini"
+  given-names: "Matteo"
+  orcid: "https://orcid.org/0000-0002-6024-861X"
+- family-names: "Gazzola"
+  given-names: "Thomas"
+  orcid: "https://orcid.org/0000-0000-0000-0000"
```
Comment (TotoGaz) [inline CITATION.cff:24]:
```suggestion
  orcid: "https://orcid.org/0000-0002-6103-4605"
```

File: CITATION.cff [hunk 11]
```diff
@@ -0,0 +1,92 @@
...
+- family-names: "Huang"
+  given-names: "Jian"
+  orcid: "https://orcid.org/0000-0002-5380-2563"
+- family-names: "Han"
+  given-names: "Brian"
+  orcid: "https://orcid.org/0000-0000-0000-0000"
+- family-names: "Gross"
+  given-names: "Herve"
+  orcid: "https://orcid.org/0000-0002-1747-2018"
+- family-names: "Mazuyer"
```
Comment (jafranc) [inline CITATION.cff:49]:
```suggestion
- family-names: "Franc"
  given-names: "Jacques"
  orcid: "https://orcid.org/0000-0002-8833-9425"
- family-names: "Mazuyer"
```

**Pull request merged**